src/classes/HouseholdService.cls

```diff
@@ -259,8 +259,13 @@ public inherited sharing class HouseholdService {
         return (Id) con.get(RECORD_TYPE_ID);
     }
 
+    private Boolean isRecordTypeInUseOnContacts;
```

Comment:
@kenlewis-sfdo  I recommend using a static var so it's retained across instances of the class.